### DISCUSSION TITLE
post/spring-oauth2-authorization-server-practice/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#21] [Announcements]
# post/spring-oauth2-authorization-server-practice/

사실상의 인가 표준(de facto)인 OAuth 2.1의 Authorization Server의 구현체를 제공하는 Spring Authorization Server 연동 방법을 공유드립니다.

https://tech.kakaopay.com/post/spring-oauth2-authorization-server-practice/

**Comment by ACACIANh** [comment]
좋은 글 감사합니다 ㅎㅎ

**Reply by traeper** [reply]
@ACACIANh OAuth 서버 개발하실 때 조금이라도 도움이 되면 좋겠습니다. 읽어주셔서 감사합니다.
